Changes

```diff
@@ -1,5 +1,16 @@
 Release history for Zonemaster component Zonemaster-CLI 
 
+v3.1.0 2021-05-28 (public release version)
+
+ [Features]
+ - Adds Finnish translation (PO file) (#192, #191)
+
+ [Fixes]
+ - Updates the installation instructions (#193, #191)
+ - Updates translations (#204, #201, #205, #200, #205,
+   #200, #207, #202)
```

Comment:
Looks like you put #200 twice. I understand that the pattern is « _#pr_number, #issue_number_ », should we duplicate _#issue_number_ if it is listed more than once ?

Reply:
Good catch. It was "#205, #200" that came double. I added it first before it was merged, and then after. Corrected.

Reply:
Oh you're right, I missed the duplication of both (good catch on my good catch ^^)